modules/toolbox/r_split.R

```diff
@@ -0,0 +1,33 @@
+
+#######   Splitter.py  ########
+
+#  Splitter is a fucntion that takes is called from python (splitter.py)
+#  The Data is read in, all the columns that are categorical are converted to factors
+#  The Data is then split into training and testing data and exported to .csv in the folder
+#  That RealSplit.py is called from.
+
+
+library(dplyr)
+
+Args <- commandArgs(trailingOnly =  T)
+print(Args[0])
+
+### reads in to csv and makes all String variables Factors #####
+
+dataset <- read.csv(as.character(Args), stringsAsFactors = T)
+
+### Randomly samples 80% of the dataset for training purposes
+train.data <- data.frame(dplyr::sample_frac(dataset, .8, replace = F))
+
+### gets the the rows from the original dataset that were not taken for the training set for testing purposes.
+test.data <- setdiff(dataset, train.data)
+
+
+### Will write to the folder calling from ####
+write.csv(test.data, "testing.data.csv")
```

Comment:
@asclines @RyanMcBerg we should specify that using this system, every dataset should have it's own folder so that there isn't conflict between different ones.